src/common/serviceUtils.js

```diff
@@ -118,3 +124,21 @@ export function handleRequestError(error) {
   logAPIErrorResponse(error);
   throw error;
 }
+
+export function isUnknownError(error) {
+  if (error.response) {
+    if (error.response.data.field_errors !== undefined) {
```

Comment:
Tiny: would it be nicer to do ```const {field_errors, errors, messages, error_code} = error.response.data;``` ?

Reply:
(Also, does data always exist for sure?)

Reply:
Yeeessssss